### DISCUSSION TITLE
UM/CF phenomenon translations: Soil Porosity?

### DISCUSSION THREAD

**Discussion by idharssi2020** [SciTools/iris#5370] [Q&A] [answered]
Hi, Can you please check the following entries in https://github.com/SciTools/iris/blob/main/lib/iris/fileformats/um_cf_map.py

    'm01s00i040': CFName('volume_fraction_of_condensed_water_in_soil_at_wilting_point', None, '1'),
    'm01s00i041': CFName('volume_fraction_of_condensed_water_in_soil_at_critical_point', None, '1'),
    'm01s00i043': CFName('soil_porosity', None, '1'),

Stash 40 and 41 seem more verbose than necessary.
Stash 43 seems to be mislabelled as porosity is not the same as volumetric soil water content at saturation (m3/m3).  
See https://www.researchgate.net/post/Is_total_water_holding_capacity_at_saturation_the_same_as_total_porosity

You could perhaps use Table 3 from this JULES paper for the descriptions https://gmd.copernicus.org/articles/4/677/2011/gmd-4-677-2011.pdf

**Comment by rcomer** [comment]
Hi @idharssi2020 good to hear from you!

The names are all taken from the [CF Standard Name table](http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html).  So unless they are incorrect, I do not think Iris would/should change them to be less verbose.  For 43, perhaps "volume_fraction_of_condensed_water_in_soil_at_field_capacity" would be more appropriate?

**Comment by idharssi2020** [comment, marked as answer]
Hi Ruth,
Unfortunately, "volume_fraction_of_condensed_water_in_soil_at_field_capacity" has a different meaning. 

"Volume_fraction_of_water_in_soil_at_saturation" would be my preference. I have opened a ticket to add this term to the CF standard name table https://github.com/cf-convention/discuss/issues/251

Should we wait and see whether the proposed term is accepted?

Thanks
Imtiaz

**Reply by ESadek-MO** [reply]
@SciTools/peloton Absolutely, the cf-convention GitHub is the place to raise this issue, if it's accepted we can pull it through to iris.
